utils/service_manager/service.go

```diff
@@ -27,20 +26,11 @@ type UIData struct {
 	Status   string
 }
 
-type ServiceData struct {
-	Command *exec.Cmd
-	FetchFn func(config.RollappConfig) ([]utils.AccountData, error)
-	UIData  UIData
-}
-
-// TODO: move this to a separate file
-// TODO: status should be Enum
-func activeIfSufficientBalance(currentBalance, threshold *big.Int) string {
-	if currentBalance.Cmp(threshold) >= 0 {
-		return "Active"
-	} else {
-		return "Stopped"
-	}
+type Service struct {
```

Comment:
next refactor will be for the `sequencer` `relayer` `da` to implement the Service interface, than u won't need to pass Fn pointers around

Add TODO for separate issue